docs/source/tutorials/python_in_mantid/plotting/03_instrument_view.rst

```diff
@@ -0,0 +1,75 @@
+.. _03_instrument_view:
+
+=======================
+Instrument View Control
+=======================
+
+#The MantidPlot instrument window viewing the GEM instrument
```

Comment:
Is this supposed to be here?

Reply:
So the instrument and sliceviewer aren't accessible from a script in workbench yet, so I copied over these pages from the old PythonInmantid course, but I haven't edited them at all! I've left them hidden from the doctree, but I guess I could be convinced to remove them. I can definitely remove any mention of MantidPlot.